README.md

```diff
@@ -100,6 +100,13 @@ $ pip3 install --user packitos
 
 You can also install packit from master branch, if you are brave enough:
 
+```
+$ dnf copr enable packit/packit-service-packit-master
```

Comment:
Small nit :D would be awesome to have a shorter name: 'packit/packit-master' - is that easy to do?

Reply:
Yes if you create the copr-repo by hand and set it in the config.

This one is the automatically-picked one.

Reply:
Do we want that?

Reply:
OK, it works:
- https://github.com/packit-service/hello-world/commits/the-build-branch
- https://copr.fedorainfracloud.org/coprs/packit/the-build-branch-test/

I am thinking if someone can misuse this.

Reply:
Updated to `packit-master`.